hazelcast/src/main/java/com/hazelcast/map/impl/MapRemoteService.java

```diff
@@ -44,8 +47,15 @@
     @Override
     public DistributedObject createDistributedObject(String name) {
         MapConfig mapConfig = nodeEngine.getConfig().findMapConfig(name);
+        checkInMemoryFormat(mapConfig.getInMemoryFormat());
 
         if (mapConfig.isNearCacheEnabled()) {
+
+            NearCacheConfig nearCacheConfig = mapConfig.getNearCacheConfig();
+            if (nearCacheConfig != null) {
```

Comment:
can this ever be false? It would mean NearCacheIsEnabled, but config is null? Is this possible?